raiden/storage/utils.py

```diff
@@ -56,10 +56,21 @@ def __getattr__(self, item):
 );
 """
 
+DB_CREATE_INVOICES = """
```

Comment:
We should store all the fields of the input and not just the encoded invoice

Reply:
So it would be saved now:

create table invoices
(
identifier INTEGER
constraint invoices_pk
primary key autoincrement,
type INTEGER,
status INTEGER,
expiration_date TEXT,
encode TEXT,
payment_hash TEXT,
secret TEXT,
currency TEXT,
amount TEXT,
description TEXT,
target_address TEXT,
token_address TEXT,
created_at TEXT
);